### DISCUSSION TITLE
Building png's from Renders

### DISCUSSION THREAD

**Discussion by Godlike-Productions** [RomanKubiak/ctrlr#194] [Show and tell]
If you want to make custom buttons or knobs, and it's beyond jKnobman, use your modelling or animation software to export a png series and you can use this website to build the images that ctrlr uses.  Just be sure to set it to vertical.

https://photo333.com/merge-png.php

**Comment by dnaldoog** [comment]
This is great - thank you!

**Comment by dobo365** [comment]
There is also the ultra powerful **ImageMagick** but with the challenge to be handled from command line only ;-)

It allows for example to morph between images (example to create 128 images from 2 images in the Orig folder that are creating 128 images auto-numbered in the New folder:
`convert Orig\*.jpg -morph 126 New\out%03d.jpg`

This command can be used to assemble our 128 images in one single vertical one:
`convert -append New\out*.jpg out.jpg`

Once you know, it looks very easy (as usual)!
